runtime/syntax/nftables.yaml

```diff
@@ -1,19 +1,25 @@
 filetype: nftables
 
 detect:
-    filename: "nftables.conf$"
+    filename: "(nftables\\.(conf|rules)$|nftables(\\.rules)?\\.d/)"
     header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
-    
+
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
-    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
-    - special: "\\b(elements|hook|policy|priority|type)\\b"
-    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
-    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
+    - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp|8021q)\\b"
+    - special: "\\b(elements|hook|policy|priority|type|state)\\b"
+    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th|dport|sport|saddr|daddr|l4proto)\\b"
+    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject|limit|queue)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "[-=/:;,@]"
-    - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
-    - constant.string: '([\"]{1})(.*)([\"]{1})'
+    - symbol.operator: "[<>.&|^!=:;,@]|\\b(and|ge|gt|le|lt|or|xor)\\b"
+      # string constants
+    - constant.string: '([\+\-\*/][\"]{1})(.*)([\"]{1})'
```

Comment:
Can you give concrete examples for this rule? Looks like it's related to operators.
Usually a string is a region in `micro`'s syntax definition.